src/components/sound.js

```diff
@@ -208,6 +219,9 @@ module.exports.Component = registerComponent('sound', {
     if (!this.loaded) {
       warn('Sound not loaded yet. It will be played once it finished loading');
       this.mustPlay = true;
+      if(processSound){
```

Comment:
single liner:
```
 if (processSound) { this.processSound = processSound; }
```

Reply:
not sure if the if statement is necessary. probably can just do `this.processSound = processSound`

Reply:
True, will do.